src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -86,7 +85,7 @@
     public ValidationStringency VALIDATION_STRINGENCY = ValidationStringency.DEFAULT_STRINGENCY;
 
     @Option(doc = "Compression level for all compressed files created (e.g. BAM and GELI).", common=true)
-    public int COMPRESSION_LEVEL = BlockCompressedStreamConstants.DEFAULT_COMPRESSION_LEVEL;
+    public int COMPRESSION_LEVEL = 1; // The default IntelDeflater is slow at levels > 1
```

Comment:
IMO end of line comments make code harder to read and edit. In most cases, like this, it can just as easily be a comment on the line before, instead of at the end of the line.